packages/core/test/integration/model/findAll.test.js

```diff
@@ -1372,6 +1372,38 @@ The following associations are defined on "Worker": "ToDos"`);
         });
       });
 
+      it('should be possible to define attributes at runtime', async function () {
+        await this.User.create();
+
+        const users = await this.User.findAll({
+          attributes: [[
+            literal('(SELECT 7 AS "X")'), // AS "X" for db2 compat
```

Comment:
@ephys something I can put to make it run for db2, using m1 chip and cannot make it run locally :(